config_manager.go

```diff
@@ -189,7 +206,7 @@ func (cm *ConfigManager) loadConfigFile() error {
 	file, err := ioutil.ReadFile(cm.Flags.ConfigFile)
 	// don't error if the default config file isn't found
 	if os.IsNotExist(err) && cm.Flags.ConfigFile == DefaultConfigFile {
-		return nil
+		return defaultConfigDoesNotExist
```

Comment:
It's totally fine for someone to only pass in params via the CLI switches and not have a config file. I'd vote for removing this warning.